packages/dd-trace/src/remote-configuration/integrity.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const crypto = require('crypto')
+const { canonicalize } = require('json-canonicalize')
+
+function findSig (signatures, rcTargetsKeyId) {
+  for (const entry of signatures) {
+    if (entry.keyid === rcTargetsKeyId) {
+      if (!entry.sig) break
+      return entry.sig
+    }
+  }
+  throw new Error(`missing signature for key ${rcTargetsKeyId}`)
+}
+
+function localAtob (str) {
+  return Buffer.from(str, 'base64').toString('ascii')
+}
+
+function toDER (hex) {
+  return Buffer.concat([
+    Buffer.from('302a300506032b6570032100', 'hex'), // Static value containing OID for algorithm
```

Comment:
Could you add more context on how you got this value?

Reply:
And also what the value does.

Reply:
@rochdev it contains the OID for the ed25519 algorithm, I will update the comment

Reply:
Updated the comment, let me know if this is clearer